target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -113,6 +113,10 @@ case "$FIRMWARE" in
 	tplink,tl-wr842n-v1)
 		caldata_extract "art" 0x1000 0x3e0
 		;;
+	ubnt,powerbridge-m|\
+	ubnt,rocket-m)
+		caldata_extract "art" 0x1000 0x1000
```

Comment:
ART size 0x1000 is suspect. If you open art partition backup file with hex editor, you will see a clear boundary between art and redundant data. And it's a good chance to convert them to nvmem format if you can test it.